src/frontend/Komponenter/Behandling/VedtakOgBeregning/Overgangsstønad/InnvilgeVedtak/InntektsperiodeValg.tsx

```diff
@@ -189,11 +196,35 @@ const InntektsperiodeValg: React.FC<Props> = ({
         }
     };
 
+    const leggTilBeregnetInntektTekstIBegrunnelse = (beregnetInntekt: {
+        årsinntekt: number;
+        minusTi: number;
+        plussTi: number;
+    }) => {
+        const beregnetTekst = `
+Forventet årsinntekt fra [DATO]: ${beregnetInntekt.årsinntekt} kroner.
+   - 10 % ned: ${beregnetInntekt.minusTi} kroner per måned.
+   - 10 % opp: ${beregnetInntekt.plussTi} kroner per måned.
+`;
+
+        const oppdatertInntektBegrunnelse = inntektBegrunnelseState.value + beregnetTekst;
+        inntektBegrunnelseState.setValue(oppdatertInntektBegrunnelse);
+    };
```

Comment:
```suggestion
        inntektBegrunnelseState.setValue((prevState) => prevState + oppdatertInntektBegrunnelse);
    };
```

For å oppdatere nåværende state med utgangspunkt i forrige state anbefaler React at du gjør det på denne måten: https://stackoverflow.com/questions/54807454/what-is-prevstate-in-reactjs

Reply:
God catch! Greit å gjøre det slik React anbefaler det 👍